org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/hotspots/HotspotsView.java

```diff
@@ -288,6 +338,68 @@ public String getText(Object element) {
     hotspotViewer.setContentProvider(ArrayContentProvider.getInstance());
   }
 
+  private void clearRule() {
+    final var NO_SECURITY_HOTSPOTS_SELECTED = "No security hotspots selected";
```

Comment:
Security Hotspots should be capitalized!

Reply:
@thahnen can you take care of this please?

Reply:
Done